committeeoversightapp/models.py

```diff
@@ -73,6 +82,21 @@ class Meta:
 
     objects = CommitteeManager()
 
+    @property
+    def hearings(self):
+        return HearingEvent.objects.all().filter(
```

Comment:
I believe you can get at related events through an Organization's `eventparticipant_set`. Is the purpose here to return `HearingEvent` objects instead of `Event` objects?

Reply:
Yes, having this be `HearingEvents` lets us use the `category` value in the hearing tables more easily.